api/src/main/java/org/openmrs/Provider.java

```diff
@@ -92,6 +96,34 @@ public String getIdentifier() {
 		return identifier;
 	}
 	
+	/**
+	 * @param role the role to set 
```

Comment:
Can you properly document all these new methods? Here is an example https://github.com/openmrs/openmrs-core/commit/f5132a1dda4a6a8fcbfe06a32dcbbe4b7a0a270d

Reply:
Addressed